styles.css

```diff
@@ -251,3 +280,193 @@ textarea {
   background-color: #2230d2;
   color: #fff;
 }
+
+@media (min-width: 600px) {
+  header {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  #nav-links {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    width: 28vw;
+    margin-right: 20px;
+  }
+
+  .logo {
+    margin-left: 53px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .intro {
+    display: flex;
+    flex-direction: column;
+    width: min-content;
+    margin: 10em 0 0 10em;
+  }
+
+  .ul {
+    display: flex;
+    width: 14vw;
+    list-style: none;
+  }
+
+  .heading #glad {
+    display: inline;
+  }
+
+  .paragraph {
+    width: 920px;
+    height: 98px;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: left;
+  }
```

Comment:
- [ ] Giving the content fixed sizes makes it non responsive, and the content will go out of it's boundaries when resized, as in the picture below. Please solve this issue by making the content width dynamic and not fixed.

![](https://i.imgur.com/xrjgdEn.png)